tests/test_strategy_commercial_producer.py

```diff
@@ -159,6 +159,7 @@ def test_event_market_cycle_does_not_create_balancing_offer_if_not_in_registry(
 
 def test_event_market_cycle_creates_balancing_offer_on_last_market_if_in_registry(
         commercial_test1, area_test1):
+    ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET = True
```

Comment:
This is also changing the global default value. Won't this affect all the tests that are run after this?

Reply:
I think one possible option would be to patch this value, either with unittest's mock.patch or with pytest (maybe [`monkeypatch`](https://docs.pytest.org/en/latest/how-to/monkeypatch.html) would be a possibility for this).

Reply:
I thought that I had a teardown method for this class, however you are right as far as I see it is missing! Adding it now and thanks for catching it